src/lib/importer/characterConverter.ts

```diff
@@ -152,13 +152,38 @@ function convertRelic(preRelic) {
 
       const subValue = subBase * count + subStep * step
 
-      substats.push({
-        stat: subStat,
-        value: Utils.precisionRound(subValue * (Utils.isFlat(subStat) ? 1 : 100), 5),
-      })
+      if (subStat.cnt != undefined && subStat.step != undefined) {
+        const rolls = { high: 0, mid: 0, low: subStat.cnt }
+
+        for (let i = 0; i < subStat.step; i++) {
+          if (rolls.low == 0) {
+            rolls.high++
+            rolls.mid--
+          } else {
+            rolls.mid++
+            rolls.low--
+          }
+        }
+        rolls.high = Math.min(subStat.cnt, rolls.high)
+        rolls.mid = Math.max(0, rolls.mid)
+        rolls.low = Math.max(0, rolls.low)
```

Comment:
Better to move these to an exported function to reuse, KelzFormatParser has a copy/paste of this